system/Database/BasePreparedQuery.php

```diff
@@ -140,11 +140,10 @@ public function prepare(string $sql, array $options = [], string $queryClass = '
 	 * The database-dependent portion of the prepare statement.
 	 *
 	 * @param string $sql
-	 * @param array  $options Passed to the connection's prepare statement.
 	 *
 	 * @return mixed
 	 */
-	abstract public function _prepare(string $sql, array $options = []);
```

Comment:
We should leave this one in for now. I don't think it's used currently, but IIRC will be necessary for other drivers in the future.